sdk/KoreBuild/modules/frameworkbuild/5-test.targets

```diff
@@ -0,0 +1,18 @@
+<Project>
+  <PropertyGroup>
+    <TestDependsOn>$(TestDependsOn);TestProjects;TestRuntimeStore</TestDependsOn>
+  </PropertyGroup>
+
+  <Target Name="TestProjects" DependsOnTargets="PrepareProjects">
+    <!-- TODO Parallel? NoBuild? -->
+    <KoreBuild
+      Projects="@(KoreBuildProject)"
+      Targets="TestProjects"
+      BuildInParallel="false"
+      Properties="$(RepoBuildProps);NoBuild=true" />
+  </Target>
+
+  <Target Name="TestRuntimeStore">
+    <!-- TODO add tests for the runtime store here -->
```

Comment:
@kichalla added tests for this by using MusicStore. We can take a look at the current CI configuration and mimic it here.